source/_integrations/motion_blinds.markdown

```diff
@@ -142,6 +142,13 @@ For tilt capable blinds a new position and tilt can be specified and the blind w
 
 ## Troubleshooting
 
+### Range of the gateway
+When the gateway is placed to far from the blinds sometimes the blinds do react to commands from the gateway, but the gateway is not able to receive messages from the blinds. In those cases the position of the blinds will not be reported correctly (both in Home Assistant and in the mobile app of the blinds). This can cause problems with specifically Top Down Bottom Up (TDBU) blinds since for TDBUs the requested position is checked to guard for collisions between the top and bottom motor, therefore not all positions are allowed based on the current position of both motors. If the gateway does not receive the current position correctly, some commands may be wrongfully rejected to prevent collisions.
```

Comment:
```suggestion

When the gateway is placed too far from the blinds, sometimes the blinds do react to commands from the gateway, but the gateway is not able to receive messages from the blinds. In those cases, the position of the blinds will not be reported correctly (both in Home Assistant and in the mobile app of the blinds). This can cause problems with specifically Top Down Bottom Up (TDBU) blinds since, for TDBUs, the requested position is checked to guard for collisions between the top and bottom motor; therefore, not all positions are allowed based on the current position of both motors. If the gateway does not receive the current position correctly, some commands may be wrongfully rejected to prevent collisions.
```